index.js

```diff
@@ -50,8 +52,24 @@ path = path.map(str => {
   return str;
 });
 
+if (typeof template === 'string') {
+  if (template.length >= 1) template = [template];
```

Comment:
it's not necessary to `else template = null;` because empty array `''` is regarded as `false` (for the purpose of [this line](https://github.com/hexojs/hexo-generator-feed/pull/110/files#diff-b58e2f1cee2acff55355343774e6d576R21)) same as `null` and `undefined`.

Empty array `[]` is regarded as `positive`, thus the necessity of [this statement](https://github.com/hexojs/hexo-generator-feed/pull/110/files#diff-168726dbe96b3ce427e7fedce31bb0bcR64).